vas2nets/tasks.py

```diff
@@ -0,0 +1,29 @@
+import csv
+import requests
+
+from django.conf import settings
+from celery.task import Task
+
+from .models import VoiceCall
+
+
+class FetchVoiceData(Task):
+
+    def get_data(self, date):
+        url = "%s?report_date=%s" % (settings.V2N_VOICE_URL, date)
+
+        content = requests.get(url).content.decode('utf-8')
+
+        return list(csv.reader(content.splitlines(), delimiter=','))[1:]
+
+    def run(self, date, **kwargs):
+        data = self.get_data(date)
+        for row in data:
+            VoiceCall.objects.create(
```

Comment:
One last thing, I'm wondering what we should do in case of duplicates? It might help to disallow duplicates for if an import fails halfway, and we need to retry it. Maybe we should change this to `get_or_create`?